.github/workflows/conda.yml

```diff
@@ -49,14 +49,17 @@ jobs:
           environment-file: devtools/conda-envs/conda_oe.yaml
           create-args: >-
             python=${{ matrix.python-version }}
+            openff-toolkit-examples
+            smirnoff-plugins=2024
+            pytest-xdist
+            pytest-rerunfailures
```

Comment:
(blocking) Could you expand on the reasoning for not putting these in `conda_oe.yaml`? Especially with version pins this seems like something that may confuse us in the future. If these are outside of that file intentionally, it'd be good to leave a comment explaining why.

Reply:
I wanted to drop that file but instead I'm just reverting all of the "conda" tests